third_party/gif_decoder/src/main/java/com/bumptech/glide/gifdecoder/GifDecoder.java

```diff
@@ -195,7 +195,7 @@ public void advance() {
      */
     public int getDelay(int n) {
         int delay = -1;
-        if ((n >= 0) && (n < header.frameCount)) {
+        if (0 <= n && n < header.frameCount) {
```

Comment:
nit: I prefer the other ordering for <= or ==, it's easier for me to read "n is greater than or equal to 0" than it is "0 is less than or equal to n".